libc/newhdrgen/yaml_combined/pthread.yaml

```diff
@@ -0,0 +1,371 @@
+      - type: int *
+  - name: pthread_attr_getguardsize
+    standards: 
+      - POSIX
+    return_type: int
+    arguments:
+      - type: const pthread_attr_t * __restrict
+      - type: size_t * __restrict
+  - name: pthread_attr_getstack
+    standards: 
+      - POSIX
+    return_type: int
+    arguments:
+      - type: const pthread_attr_t * __restrict
+      - type: void * * __restrict
+      - type: size_t * __restrict
+  - name: pthread_attr_getstacksize
+    standards: 
+      - POSIX
+    return_type: int
+    arguments:
+      - type: const pthread_attr_t * __restrict
+      - type: size_t * __restrict
+  - name: pthread_attr_setdetachstate
+    standards: 
+      - POSIX
+    return_type: int
+    arguments:
+      - type: pthread_attr_t *
+      - type: int
+  - name: pthread_attr_setguardsize
+    standards: 
+      - POSIX
+    return_type: int
+    arguments:
+      - type: pthread_attr_t *
+      - type: size_t
+  - name: pthread_attr_setstack
+    standards: 
+      - POSIX
+    return_type: int
+    arguments:
+      - type: pthread_attr_t *
+      - type: void *
+      - type: size_t
+  - name: pthread_attr_setstacksize
+    standards: 
+      - POSIX
+    return_type: int
+    arguments:
+      - type: pthread_attr_t *
+      - type: size_t
+  - name: pthread_condattr_destroy
+    standards: 
+      - POSIX
+    return_type: int
+    arguments:
+      - type: pthread_condattr_t *
+  - name: pthread_condattr_getclock
+    standards: 
+      - POSIX
+    return_type: int
+    arguments:
+      - type: const pthread_condattr_t * __restrict
+      - type: clockid_t * __restrict
+  - name: pthread_condattr_getpshared
+    standards: 
+      - POSIX
+    return_type: int
+    arguments:
+      - type: const pthread_condattr_t * __restrict
+      - type: int * __restrict
+  - name: pthread_condattr_init
+    standards: 
+      - POSIX
+    return_type: int
+    arguments:
+      - type: pthread_condattr_t *
+  - name: pthread_condattr_setclock
+    standards: 
+      - POSIX
+    return_type: int
+    arguments:
+      - type: pthread_condattr_t *
+      - type: clockid_t
+  - name: pthread_condattr_setpshared
+    standards: 
+      - POSIX
+    return_type: int
+    arguments:
+      - type: pthread_condattr_t *
+      - type: int
+  - name: pthread_create
+    standards: 
+      - POSIX
+    return_type: int
+    arguments:
+      - type: pthread_t * __restrict
+      - type: const pthread_attr_t * __restrict
+      - type: __pthread_start_t
+      - type: void *
+  - name: pthread_detach
+    standards: 
+      - POSIX
+    return_type: int
+    arguments:
+      - type: pthread_t
+  - name: pthread_equal
+    standards: 
+      - POSIX
+    return_type: int
+    arguments:
+      - type: pthread_t
+      - type: pthread_t
+  - name: pthread_exit
+    standards: 
+      - POSIX
+    return_type: void
+    arguments:
+      - type: void *
+    guard: null
+    attributes:
+      - noreturn
```

Comment:
this should go in return_type as __Noreturn void (spec.td file says to treat noreturn as a type)

Reply:
fixed!